android/src/com/frostwire/android/gui/activities/ToggleAllSearchEnginesPreference.java

```diff
@@ -60,15 +60,15 @@ public CheckBox getCheckbox() {
     @Override
     protected View onCreateView(ViewGroup parent) {
         View view = super.onCreateView(parent);
-        final TextView titleView = (TextView) view.findViewById(R.id.checkbox_preference_title);
+        final TextView titleView = (TextView) view.findViewById(R.id.view_preference_checkbox_header_title);
         titleView.setText(getTitle());
-        view.findViewById(R.id.checkbox_preference_summary).setVisibility(View.INVISIBLE);
 
-        checkbox = (CheckBox) view.findViewById(R.id.checkbox_preference_checkbox);
+        checkbox = (CheckBox) view.findViewById(R.id.view_preference_checkbox_header_checkbox);
         checkbox.setVisibility(View.VISIBLE);
         checkbox.setClickable(true);
         checkbox.setChecked(areAllEnginesChecked());
 
+
```

Comment:
does it look better now horizontally with these changes? love it when you anticipate my next fix